### PR TITLE
Hopefully a version bump fixes keycloak bugs (WIP)

### PR DESCRIPTION
**Pull request by fgeorgatos**
Hopefully a version bump will fix `../k8s-switch-limited/issues/7`

### CODE REVIEW

**Comment by fgeorgatos** [comment]
marked as (wip) since config changes might be required...

**Comment by rokroskar** [comment]
Which issues?

File: charts/renku/requirements.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ dependencies:
   version: 0.14.4
   repository: "@stable"
 - name: keycloak
-  version: 3.2.0
+  version: 4.0.1
```
Comment (leafty) [inline charts/renku/requirements.yaml]: Stage it for test

**Comment by rokroskar** [comment]
@fgeorgatos what is the status here?

File: charts/renku/requirements.yaml (hunk 1 above)
Comment (fgeorgatos) [reply charts/renku/requirements.yaml]: done; we've arrived here now, it seems that v4 has a few changes to consider here:
https://stackoverflow.com/questions/45352880/keycloak-invalid-parameter-redirect-uri

**Comment by fgeorgatos** [comment]
> @fgeorgatos what is the status here?

ongoing, tbc.

File: charts/renku/requirements.yaml (hunk 1 above)
Comment (ebiiii) [reply charts/renku/requirements.yaml]: Try with 3.4.2 and we can tag this PR with renku 0.3.0. This one shouldn't break things.

**Comment by leafty** [comment]
Postponed due to change of default signing method in newer version of keycloak

**Pull request closed without merging**